### DISCUSSION TITLE
The saving request is getting timeout

### DISCUSSION THREAD

**Discussion by pinak3748** [SoftwareBrothers/admin-bro#1440] [Q&A]
I am using the Adminjs with Prisma adapter and postgresSql as database. The Adminjs is working smoothly just when i am trying to make a edit and saving it the request is never getting fulfilled.

**Comment by pavankumar-tidke** [comment]
I'm also facing the same issue.

**Comment by nzesalem** [comment]
I had the same issue and it turned out to be one of my middleware that was causing the problem. I suggest you temporily disable your middlewares one after the other untill you find the culprit.
